vnext/Desktop/WebSocketResourceFactory.cpp

```diff
@@ -16,7 +17,7 @@ namespace Microsoft::React {
 /*static*/
 shared_ptr<IWebSocketResource>
 IWebSocketResource::Make(const string &urlString, bool legacyImplementation, bool acceptSelfSigned) {
-  if (!legacyImplementation) {
+  if (GetFeatureGate("Office.CXE.React.UseWinRTWebSocket")) {
```

Comment:
Is the `legacyImplementation` flag used anywhere now, or should it be removed?

Reply:
We haven't adopted 0.62 in Office, so changes to IWebSocketResource::Make desktop ABI should be safe.

Reply:
`legacyImplementation` can and should now be removed, but that changes the interface, so will change the interface in a future PR.